app/javascript/channels/client_channel.js

```diff
@@ -12,9 +12,8 @@ export const callback = {
   },
 
   received(data) {
-    let selector = data[0];
-    let newContent = data[1];
+    let newContent = data[0];
 
-    document.querySelector(selector).innerHTML += newContent;
+    document.querySelector(".message-list").innerHTML += newContent;
```

Comment:
.message-list is (sort of confusingly) used as a CSS className on both the notes list as well as the messages list. I wonder if it might make more sense to add some sort of custom data attribute to the dom element we want to target here.  That will create an explicit separation between classes used for styling and attributes used as query selectors for javascript. 

Maybe something like:
```
<ul class="messages-list" data-js="messages-pub-sub" />
...
document.querySelector("ul[data-js='messages-pub-sub']").innerHTML += newContent;
```